packages/styles/src/lib/Theme/index.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  createTheme as MUICreateTheme,
+  ThemeOptions,
+  Theme,
+} from '@mui/material/styles';
+import { palette as themePalette } from './palette';
+import { borderRadius as themeBorderRadius } from './borderRadius';
+import { Spacings } from './spacings';
+import { themeShadows } from './shadows';
+import { Length } from './types';
+
+export { themePalette, themeBorderRadius };
+
+export const createTheme = (
+  options?: ThemeOptions,
+  ...args: object[]
+): Theme => {
+  const {
+    components: componentsInput = {},
+    palette,
+    ...others
+  } = options ?? {};
+  return MUICreateTheme({
+    palette: {
+      ...themePalette,
+      ...palette,
+    },
+    spacing: (factor: Spacings): Length => `${Number(factor) * 0.5}rem`,
+    shadows: themeShadows,
+    components: {
```

Comment:
maybe we can create also the prop customShadow that serves the shadow token object.